Sources/AWSLambdaEvents/Codable Helpers/APIGatewayV2+Decode.swift

```diff
@@ -0,0 +1,48 @@
+#if canImport(FoundationEssentials)
+import FoundationEssentials
+#else
+import Foundation
+#endif
+
+import HTTPTypes
+
+public extension APIGatewayV2Request {
+    /// Decodes the body of the request into a `Data` object.
+    ///
+    /// - Returns: The decoded body as `Data` or `nil` if the body is empty.
+    func decodeBody() throws -> Data? {
```

Comment:
DRY
It looks like all the `decodeBody()` functions are the same. Can we create a protocol, for example `DecodableRequest` with a default implementation to avoid repeating the code twice (and maybe more if we add this to other events in the future)

Reply:
Yeah this is a great idea! Because their contents is the same, I just duplicated instead of unifying through a protocol.